web/.gitignore

```diff
@@ -1,5 +1,9 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+# local extensions & settings 
```

Comment:
I would not place local extensions & settings to global gitignore, imagine situation when each developer in team has 10 different extensions, it will become madness to maintain this file. Instead you can not place this files to stash or just not track changes